spec/javascript-spec.coffee

```diff
@@ -195,6 +195,42 @@ describe "Javascript grammar", ->
       expect(tokens[6]).toEqual value: '42', scopes: ['source.js', 'constant.numeric.js']
       expect(tokens[7]).toEqual value: ';', scopes: ['source.js', 'punctuation.terminator.statement.js']
 
+      # {tokens} = grammar.tokenizeLine('const a = 1, b = a;')
```

Comment:
Sorry, I thought I excluded this. It was a test for multiple const declarations that I couldn't get working at this time. I'll amend the commit.